.github/workflows/release.yml

```diff
@@ -0,0 +1,145 @@
+name: release
+on:
+  push:
+    tags:
+      - v*.*.*
+env:
+  # THIS GITHUB_TOKEN IS A REQUIREMENT TO BE ABLE TO WRITE TO GH RELEASES
+  GITHUB_TOKEN: ${{ secrets.RELEASE_GITHUB_TOKEN }}
+  # IF YOU NEED TO PUBLISH A NPM PACKAGE THEN ENSURE A NPM_TOKEN SECRET IS SET
+  # AND PUBLISH_NPM: TRUE. IF YOU WANT TO PUBLISH TO A PRIVATE NPM REGISTRY
+  # THEN ENSURE THE NPM_REGISTRY_URL IS CHANGED
+  # Have no idea whether this is used or not: NODE_AUTH_TOKEN: ${{ YOUR NPM TOKEN HERE }}
+  NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+  PUBLISH_NPM: true
+  NPM_REGISTRY_URL: https://registry.npmjs.org
+  # IF YOU NEED TO PUBLISH A NUGET PACKAGE THEN ENSURE AN NUGET_PUBLISH_KEY
+  # SECRET IS SET AND PUBLISH_NUGET: TRUE. IF YOU WANT TO PUBLISH TO AN ALTERNATIVE
+  # NPM REGISTRY THEN ENSURE THE NPM_REGISTRY_URL IS CHANGED
+  NUGET_PUBLISH_KEY: ${{ secrets.NUGET_PUBLISH_KEY }}
+  NUGET_FEED_URL: https://api.nuget.org/v3/index.json
+  PUBLISH_NUGET: true
+  # IF YOU NEED TO PUBLISH A PYPI PACKAGE THEN ENSURE AN PYPI_API_TOKEN
+  # SECRET IS SET AND PUBLISH_PYPI: TRUE. IF YOU WANT TO PUBLISH TO AN ALTERNATIVE
+  # PYPI REGISTRY THEN ENSURE THE PYPI_REPOSITORY_URL IS SET. IF YOU ARE USING AN API_TOKEN THEN
+  # YOU DO NOT NEED TO CHANGE THE PYPI_USERNAME (__token__) , IF YOU ARE USING PASSWORD AUTHENTICATION THEN YOU WILL
+  # NEED TO CHANGE TO USE THE CORRECT PASSWORD
+  PYPI_PASSWORD: ${{ secrets.PYPI_API_TOKEN }}
+  PYPI_USERNAME: "__token__"
+  PYPI_REPOSITORY_URL: ""
+  PUBLISH_PYPI: true
+jobs:
+  publish_binary:
+    strategy:
+      fail-fast: true
+      matrix:
+        goversion:
+          - 1.17.x
```

Comment:
Please bump to Go 1.18.x. All of Pulumi is already on that version.

Reply:
See this PR as a similar upgrade: https://github.com/pulumiverse/pulumi-time/pull/4

After changing the Go version in `go.mod` files, run `go mod tidy` for the checksum files to be regenerated.

Reply:
Thanks, solved in https://github.com/pulumiverse/pulumi-heroku/pull/4/commits/c990017b4b449e659eefcbaa877afdd4a6fd2283